### ISSUE TITLE
load_csv not showing up in litstudy=1.0.4

### ISSUE THREAD

**Issue by semmyk-research** [NLeSC/litstudy#31]
I came across issue #29 when searching for how to resolve `AttributeError: module 'litstudy' has no attribute 'load_csv'`  for `litstudy.load_csv(...)`
I see that load_csv was added in 1.0.4. This is the version installed in my `env`.
PS: I installed litstudy today with `pip install litstudy`

I went searching in my ...\envs\litstudy\lib\site-packages\litstudy folder.
I noticed `csv.py` is missing \sources.
I observed in `__init__.py` in \sources that there's no entry for `load_csv` as well.

How do I go resolving this.  Thanks.

_NB: I'm trying to use a csv file from my query in `Scholarly`_

**Comment by semmyk-research** [comment]
#walkaround as at 25 Mar '23
- download `csv.py` from repository
- edit `__init__.py` to include entry for `load_csv`

In my working file (JupyterLab)
```python
import litstudy.sources  ##load subfolder with __init__
#from litstudy import *  #load_csv

## reload library
from importlib import reload  # Python 3.8.16
reload(litstudy.sources)
```

**Comment by stijnh** [comment]
Thanks for using litstudy!

This was fixed by c669fab and `load_csv` is available in litstudy 1.0.5
